larray_editor/view.py

```diff
@@ -1627,7 +1626,9 @@ def setup_and_check(self, arrays, names, title=''):
         Setup ArrayComparator:
         return False if data is not supported, True otherwise
         """
-        assert all(isinstance(a, la.LArray) for a in arrays)
+        if not all(isinstance(a, la.LArray) for a in arrays):
+            QMessageBox.critical(self, "Error", "Expected only LArray objects in argument 'arrays'. "
+                                                "Got {}".format(', '.join([type(a) for a in arrays])))
```

Comment:
besides, I am unsure this assert should be converted to an alert because we are still in API-land here (this is not called by any action within the GUI).